posthog/admin/admins/user_admin.py

```diff
@@ -98,19 +107,23 @@ def current_organization_link(self, user: User):
             user.organization.name,
         )
 
+    @admin.display(description="Web sessions")
+    def revoke_sessions_link(self, user: User):
+        return format_html('<a href="{}" class="button" id="revoke_sessions_button">{}</a>', "#", "Revoke all")
```

Comment:
I would ideally like to also display the number of active sessions, but the lookup is fairly expensive, so it seemed best not to add this yet.